src/snapred/backend/log/logger.py

```diff
@@ -59,10 +60,26 @@ def _setFormatter(self, logger):
         logger.addHandler(ch)
         return logger
 
+    def _warning(self, message, vanillaWarn):
```

Comment:
does this mean users have to call `logger._warning()` instead of `logger.warning()`?

Reply:
No, the warn/warning methods get replaced a little further down using this in a lambda.